crates/ndc-graphql-cli/Cargo.toml

```diff
@@ -9,6 +9,7 @@ common = { path = "../common" }
 graphql_client = "0.14.0"
 graphql-introspection-query = "0.2.0"
 graphql-parser = "0.4.0"
+ndc-models = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.4" }
```

Comment:
Is there any reason we are not on `v0.1.6`?

Reply:
We need ndc-models to be at the same version as the ndc-sdk uses.
I'm updating the sdk so I can also update the models